devtools-html/debugger.html
# [CallStack] Simplify call stacks involving Babel async transforms

Associated Issue: #5150 

### Summary of Changes
* Annotate frames with certain displayNames or URLs as being Babel frames, so that frames related to async transforms get grouped together.

### Test Plan
1. Go to http://firefox-dev.tools/debugger-examples/examples/babel/
2. Set breakpoints in `entry.js` lines 28 and 31.
3. Hit the `asyncExample` button.
4. On either breakpoint, frames related to async transforms should be grouped together as being Babel frames. (see gif below)

Regression test:
1. Go to http://firefox-dev.tools/debugger-examples/examples/my-app/build/
2. Set a breakpoint on App.js#14
3. Grouping for React and Webpack frames should work like before.

### Screenshots/Videos
![babelcallstack](https://user-images.githubusercontent.com/7321311/36616406-a40163a2-18b1-11e8-8d3f-cef044e7884a.gif)
